lm_eval/tasks/superglue.py

```diff
@@ -80,29 +83,37 @@ def doc_to_text(self, doc, include_target=True):
             # Neither = neutral
             text += " {}".format({0: "true", 1: "neither", 2: "false"}[doc["label"]])
```

Comment:
Don't include target in doc_to_text return (delete these lines)

Reply:
Merged the code that removed these